server/src/main/java/org/opensearch/index/shard/IndexShard.java

```diff
@@ -1377,6 +1384,12 @@ public MergeStats mergeStats() {
     public SegmentsStats segmentStats(boolean includeSegmentFileSizes, boolean includeUnloadedSegments) {
         SegmentsStats segmentsStats = getEngine().segmentsStats(includeSegmentFileSizes, includeUnloadedSegments);
         segmentsStats.addBitsetMemoryInBytes(shardBitsetFilterCache.getMemorySizeInBytes());
+        // Populate remote_store stats only if the index is remote store backed
```

Comment:
We are doing an add here and one explicit add in NoOpEngine.java class -
https://github.com/opensearch-project/OpenSearch/blob/4871e25ab7f5fac5b3c2ec19d937dad54e5a2a81/server/src/main/java/org/opensearch/index/engine/NoOpEngine.java#L140
This seem to be adding same stats twice.